hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/pipeline/ListPipelinesSubcommand.java

```diff
@@ -39,23 +41,41 @@
     versionProvider = HddsVersionProvider.class)
 public class ListPipelinesSubcommand extends ScmSubcommand {
 
-  @CommandLine.Option(names = {"-ffc", "--filterByFactor"},
-      description = "Filter listed pipelines by Factor(ONE/one)",
+  @CommandLine.Option(names = {"-t", "--type"},
+      description = "Filter listed pipelines by replication type, RATIS or EC",
       defaultValue = "")
-  private String factor;
+  private String replicationType;
 
-  @CommandLine.Option(names = {"-fst", "--filterByState"},
-      description = "Filter listed pipelines by State(OPEN/CLOSE)",
```

Comment:
Can we keep these old options for backward compatibility?

Reply:
I can add them back in, but in my opinion we shouldn't. The current options don't fit with the convention of type / replication used everywhere else and why do we even have "ffc" for "filter by factor"? 

I am all for compatibility on important areas, but this is for a likely rarely used command that is not critical for any operations on the cluster. The extra options will make the help text look messy and we will end up carrying around 4 ways to pass the factor forever.

Reply:
Thanks for restoring them.  This command may look like rarely used, but it is useful for automated testing (validating cluster state).

I agree that these old options are non-standard (and clumsy), and would like to get rid of them eventually.  But we should deprecate them first.  Co-existing old and new options allow non-breaking upgrade, giving time to update test scripts.  If we simply replace them, downstream users need to adapt at the same time as bringing in these Ozone changes, which can be a chicken-egg problem.